### PR TITLE
Cta paid content fixes

### PR DESCRIPTION
**Pull request by Calanthe**
Minor changes to the 'paid content' cta text and glabs logo links (different link for the Australian edition).

cc @jbreckmckye @kelvin-chappell

### CODE REVIEW

File: common/app/views/fragments/containers/facia_cards/commercialContainer.scala.html [hunk 1]
```diff
@@ -24,13 +25,19 @@
                                 <div class="paidfor-label paidfor-meta__more has-popup">
                                     <button class="u-button-reset paidfor-label__btn popup__toggle" data-toggle="paidfor-popup--@container.dataId">About @inlineSvg("arrow-down", "icon")</button>
                                     <div class="popup is-off paidfor-popup paidfor-popup--@container.dataId">
-                                        <h3 class="paidfor-popup__title">Paid stories are paid for and controlled by an advertiser</h3>
-                                        <a class="paidfor-popup__link" href="#">Learn more about the Guardian’s funding from outside parties @inlineSvg("arrow-right", "icon")</a>
+                                        <h3 class="paidfor-popup__title">Paid stories are paid for and approved by an advertiser and created by the Guardian Labs commercial editorial team</h3>
+                                        <a class="paidfor-popup__link" href="https://www.theguardian.com/info/2014/sep/23/paid-for-content">Learn more about Guardian Labs content @inlineSvg("arrow-right", "icon")</a>
```
Comment (jbreckmckye) [inline common/app/views/fragments/containers/facia_cards/commercialContainer.scala.html]: Is there a helper to inject the domain? That way it'd point to the m.code page on m.code, the localhost page on localhost, etc.

**Comment by jbreckmckye** [comment]
Looks good to me :+1:

File: common/app/views/fragments/containers/facia_cards/commercialContainer.scala.html (hunk 1 above)
Comment (kelvin-chappell) [inline common/app/views/fragments/containers/facia_cards/commercialContainer.scala.html]: `@LinkTo(/info/2014/sep/23/paid-for-content)`

**Comment by kelvin-chappell** [comment]
:+1:

**Pull request merged**